modules/miscutil/lib/upgrades/invenio_2014_09_09_tag_recjsonvalue_not_null.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2014 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+import warnings
+from invenio.dbquery import run_sql
+from invenio.textutils import wait_for_user
+
+depends_on = ['invenio_2014_08_13_tag_recjsonvalue']
+
+
+def info():
+    """Upgrade recipe information."""
+    return "tag.recjson_value NOT NULL"
+
+
+def do_upgrade():
+    """Upgrade recipe procedure."""
+    run_sql(
+        "ALTER TABLE tag CHANGE COLUMN recjson_value recjson_value text NOT NULL;")
```

Comment:
I'll amend this recipe during merge to also do:

```
update tag set recjson_value='' where recjson_value is null;
```

otherwise people upgrading from v1.1.5 might be bitten by this...